js/serial.js

```diff
@@ -1,6 +1,9 @@
 'use strict';
 
 var serial = {
+    connectionType:  'serial', // 'serial' or 'tcp'
```

Comment:
Just for maintainability, can you please add new stuff to the bottom of the list?